src/components/tutorials-sidebar/components/tutorial-view-sidebar-content/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { useMemo } from 'react'
+// Hooks
+import { useCollectionProgress } from 'hooks/progress'
+import { useTutorialProgress } from 'hooks/progress'
+// Components
+import {
+	CollectionProgressStatusSection,
+	parseCollectionProgress,
+} from 'components/collection-progress-group'
+import TutorialProgressIcon from 'components/tutorial-progress-icon'
+import { SidebarNavMenuItem } from 'components/sidebar/components'
+import { SectionList } from 'components/tutorials-sidebar'
+// Types
+import { TutorialListItemProps } from 'components/tutorials-sidebar/types'
+// Styles
+import s from './tutorial-view-sidebar-content.module.css'
+import { Collection } from 'lib/learn-client/types'
+
+/**
+ * Renders sidebar content for tutorial views.
+ */
+function TutorialViewSidebarContent({
+	collection,
+	items,
+}: {
+	collection: Collection
+	items: TutorialListItemProps[]
+}) {
+	return (
+		<>
+			<CollectionProgress collection={collection} />
+			<SectionList>
+				{items.map((item: TutorialListItemProps) => {
+					return (
+						<TutorialListItem
+							key={`${item.collectionId}${item.tutorialId}`}
+							text={item.text}
+							href={item.href}
+							isActive={item.isActive}
+							tutorialId={item.tutorialId}
+							collectionId={item.collectionId}
+						/>
+					)
+				})}
+			</SectionList>
+		</>
+	)
+}
+
+/**
+ * Displays collection progress status.
+ */
+function CollectionProgress({ collection }: { collection: Collection }) {
```

Comment:
I've split this work out of the `tutorials-sidebar` index file, since it was getting big, and this felt more clear.

The main change other than this move is the addition of `CollectionProgress`, which uses `CollectionProgressStatusSection` and `parseCollectionProgress` from `components/collection-progress-group`.